rpc/rpc_machine.c

```diff
@@ -468,12 +470,13 @@ static int rpc_tm_setup(struct m0_net_transfer_mc *tm,
 	if (rc < 0)
 		return M0_ERR_INFO(rc, "TM initialization");
 
+	if (net_dom->nd_xprt == &m0_net_lnet_xprt)
+		max_msgs_size = m0_rpc_max_msg_size(net_dom, msg_size);
+	else
+		max_msgs_size = 4096; //TODO should be confirm by Nikita
```

Comment:
I am sorry, I was wrong.
This might be set to a very large number, like 0xffffffff.
LNET limits the max_msgs_size to 1MB, but this is not a limit for sock.

Anyway, let's ask Nikita for his comments.

Please use /* */ style of comments.